### DISCUSSION TITLE
How do I set a page title in the app directory, if I am using "use client"?

### DISCUSSION THREAD

**Discussion by vedantroy** [vercel/next.js#49183] [App Router]
I can't use the metadata API to set page metadata in the root layout, since I'm using "use client".
At the same time, using the `next/head` component to set page metadata is not working either.

**Comment by icyJoseph** [comment]
Is it not possible at all to refactor your layout so that the `layout.tsx` is not `'use client';`?

**Reply by nick-kang** [reply]
'use server' doesn't work with static export, which is our preferred method of deployment for sites with static data (eg, Marketing Site, Blogs).

**Comment by sharfudeen** [comment]
Issue resolved? If so, please post it here.

**Reply by icyJoseph** [reply]
You just have to refactor:

From:

```tsx
// app/page.tsx
'use client'

export default function Page(){
  // page code
}
```

To

```tsx
// app/page.tsx
import {Home} from './home'

// export meta-data

export default function Page(){
  return <Home />
}
```

Where:

```tsx
// app/home.tsx

export function Home(){
  // page code
}
```
